docs/content/en/docs/yaml-crd-ref/evaluationdefinition.md

```diff
@@ -0,0 +1,79 @@
+---
+title: KeptnEvaluationDefinition
+description: Define all evaluations associated with an application
+weight: 30
+---
+
+A `KeptnEvaluationDefinition` defines evaluation tasks
+that can be run by the Keptn Lifecycle Toolkit
+as part of pre- and post-analysis phases of a workload or application.
+
+## Yaml Synopsis
+
+```yaml
```

Comment:
with v1alpha3, the structure of the `KeptnEvaluation`s has changed, as they don't include the query anymore, but rather refer the a `KeptnMetric` that contains the query. An example of a `KeptnEvaluationDefinition` now looks as follows:

```
apiVersion: lifecycle.keptn.sh/v1alpha3
kind: KeptnEvaluationDefinition
metadata:
  name: pre-deployment-hello
spec:
  objectives:
    - evaluationTarget: ">1"
      keptnMetricRef:
        name: available-cpus 
        namespace: some-namespace
```

The referenced KeptnMetric then contains the `query` that was previously present in the `KeptnEvaluationDefinition`:

```
apiVersion: metrics.keptn.sh/v1alpha2
kind: KeptnMetric
metadata:
  name: available-cpus
  namespace: some-namespace
spec:
  fetchIntervalSeconds: 10
  provider:
    name: prometheus
  query: >-
    sum(kube_node_status_capacity{resource='cpu'})
```

Reply:
Thanks!  I think this is incorporated now